workers/unity/Packages/com.improbable.gdk.core/Logging/ForwardingDispatcher.cs

```diff
@@ -44,7 +44,7 @@ private void LogCallback(string message, string stackTrace, LogType type)
             // This is required to avoid duplicate forwarding caused by HandleLog also logging to console
             if (type == LogType.Exception)
             {
-                Connection?.SendLogMessage(LogLevel.Error, Connection.GetWorkerId(), $"{message}\n{stackTrace}");
+                Worker.Connection?.SendLogMessage(LogLevel.Error, Worker.Connection.GetWorkerId(), $"{message}\n{stackTrace}");
```

Comment:
Can `Worker` be null here?

Reply:
if you create a dispatcher and use it directly instead of giving it to the worker: yes